Market-API.txt

```diff
@@ -9,7 +9,13 @@ The current on that is:
 
 from tribler/Tribler/community/market/community.py
 
-Division of responsibility: the market is fully responsible for the exchange of funds and management of the wallets of both Multi-chain, Bitcoin.
+Division of responsibility: 
+The Multi-chain database, wallet and money transfer mechanism is fully under control of The Market team.
+A get_balance() function for multi-chain needs to be created by either: one of the BEP teams of dev team of multi-chain.
```

Comment:
We have a get balance function, but it is not easily accessible for other parties. But you can have a look at that